src/main/java/springboot/repository/impl/BookRepositoryImpl.java

```diff
@@ -1,47 +1,47 @@
 package springboot.repository.impl;
 
+import java.util.List;
+import org.hibernate.Session;
 import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
 import org.hibernate.query.Query;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
 import springboot.exeptions.DataProcessingException;
 import springboot.model.Book;
 import springboot.repository.BookRepository;
-import org.hibernate.Session;
-import org.hibernate.Transaction;
-import org.springframework.stereotype.Repository;
-
-import java.util.List;
 
 @Repository
```

Comment:
```suggestion
@Repository
@RequiredArgsConstructor
```

Reply:
import org.hibernate.Transaction;
import org.springframework.stereotype.Repository;
import java.util.List;

without this imports i can't implements save method